Draft event page - july comm call (#586)
ropensci/roweb3: content/commcalls/2023-07-31/index.md

---
title: "Mentoring & training program for Scientific Open Source Champions"
dateStart: 2023-07-31T20:00:00 # UTC!!
dateEnd: 2023-07-31T21:00:00 # UTC!!
date: 2023-07-31T21:00:00 # UTC!! same as dateEnd
publishDate: 2023-06-30
description: Details about the Champions Program + tips for submitting your application + the experience of being an rOpenSci champion or mentor
author:
  - Beatriz Milz
  - Carolina Pradier
  - Victor Ordu
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: july2023-championprogram
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:

>   - url: 

Author: ```suggestion
  - url: https://github.com/BroVic/naijR
```

>     name:   

Author: ```suggestion
    name:   R package on Nigeria and for Nigeria 
```

>     details: 

Author: ```suggestion
    details: The goal of naijR is to make it easier for R users to work with data related to Nigeria. This package is going through the rOpenSci software peer review. 
```

speakers:  
  - name: Beatriz Milz
    bio: 
    img: /img/community/picture.jpeg

>   - name: 

Author: ```suggestion
  - name: Victor Ordu
```

    bio:        
    img: /img/community/picture.jpg    

>   - name:  

Author: ```suggestion
  - name:  Carolina Pradier
```

> 

Author: ```suggestion
resources:
  - url: https://github.com/holatam/eph
    name:  eph - Toolbox for the processing of the Permanent Household Survey in Argentina
    details: package to download and manipulate the Permanent Household Survey from Argentina (eph is the Spanish acronym for Permanent Household Survey)
```

---
title: "Mentoring & training program for Scientific Open Source Champions"

> dateStart: 2023-07-31T20:00:00 # UTC!!

Author: ```suggestion
dateStart: 2023-07-25T14:00:00 # UTC!!
```

> dateEnd: 2023-07-31T21:00:00 # UTC!!

Author: ```suggestion
dateEnd: 2023-07-25T15:00:00 # UTC!!
```

> date: 2023-07-31T21:00:00 # UTC!! same as dateEnd

Author: ```suggestion
date: 2023-07-31T15:00:00 # UTC!! same as dateEnd
```

publishDate: 2023-06-30
description: Details about the Champions Program + tips for submitting your application + the experience of being an rOpenSci champion or mentor
author:
  - Beatriz Milz
  - Carolina Pradier
  - Victor Ordu
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: july2023-championprogram
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:
  - url: https://github.com/BroVic/naijR
    name:   R package on Nigeria and for Nigeria 
    details: The goal of naijR is to make it easier for R users to work with data related to Nigeria. This package is going through the rOpenSci software peer review. 
  - url: https://github.com/holatam/eph
    name:  eph - Toolbox for the processing of the Permanent Household Survey in Argentina
    details: package to download and manipulate the Permanent Household Survey from Argentina (eph is the Spanish acronym for Permanent Household Survey)

> speakers:  

Author: ```suggestion
  - url: /champions/
    name:   Champions Program
    details: web page with details about the champion program.
speakers:  
```

  - name: Beatriz Milz

>     bio: I'm a mentor in the rOpenSci's Champions Program . I'm an R-Lady from São Paulo - Brazil and a Ph.D. Candidate in Environmental Sciences at University of São Paulo and I also work as a Data Scientist and an R teacher at Curso-R. I am a tidyverse instructor certified by Posit, and a co-organizer of communities and events, such as Latin-R, and satRday São Paulo.

Reviewer: I think would be good to cite that I'm a R-Ladies Co-organizer.
```suggestion
    bio: I'm a mentor in the rOpenSci's Champions Program. I'm an R-Ladies co-organizer in São Paulo - Brazil and a Ph.D. Candidate in Environmental Sciences at University of São Paulo. I also work as a Data Scientist and an R teacher at Curso-R. I am a tidyverse instructor certified by Posit, and a co-organizer of communities and events, such as Latin-R, R-Ladies São Paulo and satRday São Paulo.
```

    img: /img/team/beatriz-milz-mentor.jpeg
  - name: Victor Ordu

>     bio: I'm a medical doctor with many years of experience in the Nigerian healthcare space. I'm an independent consultant for the UN and international NGOs working in Nigeria, R has been the focus of much of my work. I also conducts R training programs for public and private organizations in Nigeria and I'm the author of R packages, one of them is part of my project as rOpenSci Champion.        

Author: ```suggestion
    bio: I'm a medical doctor with many years of experience in the Nigerian healthcare space. I'm an independent consultant for the UN and international NGOs working in Nigeria, and R has been the focus of much of my work. I also conduct R training programs for public and private organizations in Nigeria. I'm the author of R packages; one is part of my project as rOpenSci Champion.        
```

    img: /img/team/victor-ordu-mentee.jpg  
  - name:  Carolina Pradier
    bio:  I am an economist and data scientist. I have a degree in Economics, and I am currently finishing my Master’s degree in Labour Studies, both at the University of Buenos Aires. I work as a research and teaching assistant at CEPED-UBA and I’m also an active member of Ecofeminita’s economics and data team (ecofemidata) and maintainer of the eph R package, which is my project as rOpenSci Champion. 
    img: /img/team/carolina-pradier-mentee.jpg  
tags:
  - community call
  - events
  - champions program
deets: |

>     Meeting ID:  

Author: ```suggestion
    Meeting ID:  875-494-77466
```

>     Passcode: 

Author: ```suggestion
    Passcode: 516675
```

> zoomurl: 

Author: ```suggestion
zoomurl: https://numfocus-org.zoom.us/j/87549477466?pwd=b0cydVpIYW5yMm9MKzZONG16RTVsUT09
```

---
title: "Mentoring & training program for Scientific Open Source Champions"
dateStart: 2023-07-31T20:00:00 # UTC!!
dateEnd: 2023-07-31T21:00:00 # UTC!!
date: 2023-07-31T21:00:00 # UTC!! same as dateEnd
publishDate: 2023-06-30
description: Details about the Champions Program + tips for submitting your application + the experience of being an rOpenSci champion or mentor
author:
  - Beatriz Milz
  - Carolina Pradier
  - Victor Ordu
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: july2023-championprogram
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
# - id: 
resources:
  - url: https://github.com/BroVic/naijR
    name:   R package on Nigeria and for Nigeria 
    details: The goal of naijR is to make it easier for R users to work with data related to Nigeria. This package is going through the rOpenSci software peer review. 

> resources:

Author: ```suggestion

```
